scripts/benchmark.ts

```diff
@@ -33,6 +34,7 @@ async function main() {
 
     const tx = await batch.submit(
         constracts.rollup,
+        batchID,
         TESTING_PARAMS.STAKE_AMOUNT
```

Comment:
What do you think we add the expected batchID as a property of the batch class? So that the submit function automatically fills in that value.
It can be an undefined property by default, and the submit raise an error if it is unset. 
But feel free to ignore this comment as we are replacing the batch class used in the contract tests with the one that the client use.

Reply:
I think it would be better to wait until we switch. I can definitely see adding the batch ID to the newer `Batch` interface, could also make the `BatchStorage` implementation simpler.

Reply:
wait until we switch sounds good